packages/postgres-database/src/simcore_postgres_database/models/users.py

```diff
@@ -36,11 +45,14 @@ class UserRole(Enum):
     TESTER = "TESTER"
     ADMIN = "ADMIN"
 
-    @classmethod
-    def super_users(cls):
-        return list(itertools.takewhile(lambda e: e != cls.USER, cls))
+    @property
+    def privilege_level(self) -> int:
+        return _USER_ROLE_TO_LEVEL[self.name]
```

Comment:
not sure how this works. so this function returns 100 if I am admin? so you check for 100 afterwards?
not sure that is very clear

Reply:
also the other function was also unclear actually.